### ISSUE TITLE
IntAct Adapter

### ISSUE THREAD

**Issue by bnymnsen** [saezlab/BioCypher#123]
Part of #33 .

Link: https://github.com/HUBioDataLab/CROssBAR-BioCypher-Migration/blob/main/bccb/intact_adapter.py

**Comment by bnymnsen** [comment]
@slobentanzer I added node and edge type information to the adapters. Can you think of any other useful information that could be added on issue description of adapters?

**Comment by slobentanzer** [comment]
Hi @bnymnsen, I would keep free-text entries at a minimum. What I would like is to get to an automatable representation of all adapters and pipelines with little possibility for error (typos and such).

Did you have a look at the meta-graph repository? You can build a docker Neo4j version of the meta-graph locally, which is populated by the GitHub API from these issues. The only text processing on the description I do is currently the association between adapters and their pipelines (the issues being mentioned in the pipeline description). So all additional things should be easy to do **consistently**, which is why for most attributes I have made a field on the project board (input type, adapter granularity, etc). If you can think of additions to that process (we can create new fields in various formats), let me know. :) We can also discuss on Zulip in a dedicated topic.

**Comment by bnymnsen** [comment]
No, I didn't. I will look and run it ASAP :). Frankly, I am a bit confused, but I guess I will understand better when I run and see it. After, we can discuss it on Zulip chat.
